src/main.cpp

```diff
@@ -5430,10 +5489,6 @@ bool CVerifyDB::VerifyDB(const CChainParams& chainparams, CCoinsView *coinsview,
             }
         }
 
-        if (!orchardAuth.Validate()) {
```

Comment:
Where do Orchard signatures get validated for DB verification now?

Reply:
They don't. We were only doing this here because we needed to pass an Orchard validator into the `CheckBlock` API, so we figured we may as well use the batched signatures. Now that we don't need to pass it into `CheckBlock`, we don't bother re-validating these signatures. Note that the existing code from Bitcoin did not re-validate transparent signatures here.